collections/_infographics/policies/emission-pledges.md

```diff
@@ -12,4 +12,34 @@ data-orig:
   - [ "Our Jupyter notebook", "https://github.com/faktaoklimatu/data-analysis/blob/master/notebooks/net-zero-pledges.ipynb" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emisni-zavazky" %}
+## How to read an infographic?
+{% comment %}
+{% include preview-box.html
+    title="What is carbon neutrality?"
+    text="We explain exactly what the zero carbon commitment means and how the term can be confusing."
+    slug="uhlikova-neutralita"
+%}
+{% endcomment %}
+This infographic shows the share of carbon-neutral countries in global CO<sub>2</sub> emissions in 2018. Excluding the Land Use, Land Use Change and Forestry sector (*LULUCF), all CO<sub>2</sub> emissions, including international maritime and aviation transport. **As of March 2022, 87% of global emissions come from countries moving towards carbon neutrality**, with 131 countries total.
+
+Countries that are moving towards carbon neutrality are those that have commitments: fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage. If we exclude countries with commitments in the draft or discussion phase, there would be a 3% reduction in covered emissions to 84% of global emissions.
+
+Some countries claim to be climate neutral (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*). However, we do not distinguish between them in the infographic and use the term carbon neutrality for all commitments. While the former refers to all greenhouse gases, the latter refers "only" to CO<sub>2</sub>. I.e., if a country commits to climate neutrality, this commitment includes carbon neutrality. <!--For more information on carbon neutrality in general, see our [explainer](/explainers/carbon-neutrality).--> Countries worldwide must achieve carbon neutrality by around 2050 to keep global warming below 1.5°C. Commitments towards later dates are insufficient to meet this goal.
```

Comment:
rephrase: "Some countries claim to be climate neutral (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*).  However, we do not distinguish between them in the infographic and use the term carbon neutrality for all commitments."

as "Some countries aim for climate neutrality (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*). We do not distinguish between these two terms in this infographic and use the term carbon neutrality for all commitments."

Reply:
Thanks for catching this!